src/components/InterfaceEditor.tsx

```diff
@@ -272,7 +272,7 @@ const checkInterfaceHasMajorChanges = (
     );
     return _.isEqual(draftMapping, initialMapping);
   });
-  if (!hasExistingMappings) {
+  if (!hasExistingMappings && draftInterface.minor <= initialInterface.minor) {
```

Comment:
It would be good to add a cypress test to make sure the features works as intended